bin/cqlsh.py

```diff
@@ -2091,7 +2095,6 @@ def is_file_secure(filename):
     # This is to allow "sudo cqlsh" to work with user owned credentials file.
     return (uid == 0 or st.st_uid == uid) and stat.S_IMODE(st.st_mode) & (stat.S_IRGRP | stat.S_IROTH) == 0
 
-
```

Comment:
@bhouse99 this line should not be removed. I think there is 2 lines policy in Python between methods.